test/jdk/java/foreign/TestBufferStackStress2.java

```diff
@@ -23,6 +23,11 @@
 
 /*
  * @test
+ *
+ * @comment If the VM does not have continuations, then VTs will be scheduled on OS threads.
+ * @requires vm.continuations
+ *
+ * @bug 8356114, 8356658
```

Comment:
jtreg doesn't like the presence of comma in those values and the GitHub job has caught that failure in this test.